pyvista/__init__.py

```diff
@@ -45,6 +46,11 @@
 except KeyError:
     pass
 
+# If available, a local vtk-data instance will be used for examples
+VTK_DATA_PATH: Optional[str] = None
+if 'PYVISTA_VTK_DATA' in os.environ:
+    VTK_DATA_PATH = os.environ['PYVISTA_VTK_DATA']
```

Comment:
Optionally check if this is a valid path here.  Might clear up some headaches when debugging this.

Reply:
I will add a warning when it doesn't match in `pyvista.__init__`.  This seems better than not allowing pyvista to import by raising an error.

Reply:
Should we perhaps raise a loud error here instead? Someone set the envvar for a reason, and they might miss the warning (perhaps in CI?). What kind of use cases are we preparing for with this feature? If "silently" (with a warning) downloading is OK considering those use cases, then just a warning is fine as well.

Reply:
I'm okay with raising an error here too. I don't have a strong feeling.

We could also warn on import and raise an error with an understandable error message when the user tries to download.  This seems like the best of both worlds to me.